pymedphys/_experimental/pinnacle/rtdose.py

```diff
@@ -338,6 +342,14 @@ def convert_dose(plan, export_path):
         idx = [0.0, 0.0, 0.0]
         for i in range(3):
             idx[i] = -(origin[i] - prescription_point[i]) / spacing[i]
+
+        if patient_position in ("HFP", "FFS"):
+            idx[0] = -idx[0]
+        if patient_position in ("HFP", "FFP"):
+            idx[1] = -idx[1]
+        if patient_position in ("FFS", "FFP"):
+            idx[2] = -idx[2]
+
```

Comment:
Is there a way to implement this logic in a general way that applies to all orientations by creating a transform from by `ds.ImageOrientationPatient`?